Index: template/material-dashboard-angular2-master/src/app/user-profile/user-profile.component.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- template/material-dashboard-angular2-master/src/app/user-profile/user-profile.component.html	(revision 4a92d4a716e01abdfb387e01dd08072e9663dd94)
+++ template/material-dashboard-angular2-master/src/app/user-profile/user-profile.component.html	(date 1584620114965)
@@ -90,7 +90,7 @@
                     </div>
                 </div>
             </div>
-            <div class="col-md-4">
+           <!-- <div class="col-md-4">
                 <div class="card card-profile">
                     <div class="card-avatar">
                         <a href="javascript:void(0)">
@@ -106,7 +106,7 @@
                         <a href="javascript:void(0)" class="btn btn-danger btn-round">Follow</a>
                     </div>
                 </div>
-            </div>
+            </div>-->
         </div>
     </div>
 </div>
Index: template/material-dashboard-angular2-master/src/app/app.module.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- template/material-dashboard-angular2-master/src/app/app.module.ts	(revision 4a92d4a716e01abdfb387e01dd08072e9663dd94)
+++ template/material-dashboard-angular2-master/src/app/app.module.ts	(date 1584618845914)
@@ -13,6 +13,10 @@
 import { DashboardComponent } from './dashboard/dashboard.component';
 import { UserProfileComponent } from './user-profile/user-profile.component';
 import { TableListComponent } from './table-list/table-list.component';
+import { AddRecruteursComponent } from './add-recruteur/add-recruteurs.component';
+import { ListRecruteursComponent } from './list-recruteurs/list-recruteurs.component';
+import { AddResprhComponent } from './add-resprh/add-resprh.component';
+import { ListResprhComponent } from './list-resprh/list-resprh.component';
 import { TypographyComponent } from './typography/typography.component';
 import { IconsComponent } from './icons/icons.component';
 import { MapsComponent } from './maps/maps.component';
@@ -22,11 +26,8 @@
   AgmCoreModule
 } from '@agm/core';
 import { AdminLayoutComponent } from './layouts/admin-layout/admin-layout.component';
-import { AddRecruteursComponent } from './add-recruteur/add-recruteurs.component';
-import { AddResprhComponent } from './add-resprh/add-resprh.component';
+
 import {MatFormFieldModule} from "@angular/material/form-field";
-import { ListRecruteursComponent } from './list-recruteurs/list-recruteurs.component';
-import { ListResprhComponent } from './list-resprh/list-resprh.component';
 
 @NgModule({
   imports: [
@@ -45,10 +46,10 @@
   declarations: [
     AppComponent,
     AdminLayoutComponent,
-    AddRecruteursComponent,
-    AddResprhComponent,
-    ListRecruteursComponent,
-    ListResprhComponent,
+    //AddRecruteursComponent,
+    //AddResprhComponent,
+    //ListRecruteursComponent,
+    //ListResprhComponent,
 
   ],
   providers: [],
Index: template/material-dashboard-angular2-master/src/app/layouts/admin-layout/admin-layout.module.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- template/material-dashboard-angular2-master/src/app/layouts/admin-layout/admin-layout.module.ts	(revision 4a92d4a716e01abdfb387e01dd08072e9663dd94)
+++ template/material-dashboard-angular2-master/src/app/layouts/admin-layout/admin-layout.module.ts	(date 1584619162604)
@@ -7,6 +7,9 @@
 import { UserProfileComponent } from '../../user-profile/user-profile.component';
 import { TableListComponent } from '../../table-list/table-list.component';
 import { AddRecruteursComponent } from '../../add-recruteur/add-recruteurs.component';
+import { ListRecruteursComponent } from '../../list-recruteurs/list-recruteurs.component';
+import { AddResprhComponent } from '../../add-resprh/add-resprh.component';
+import { ListResprhComponent } from '../../list-resprh/list-resprh.component';
 //import { TypographyComponent } from '../../typography/typography.component';
 //import { IconsComponent } from '../../icons/icons.component';
 //import { MapsComponent } from '../../maps/maps.component';
@@ -36,7 +39,10 @@
     //DashboardComponent,
     UserProfileComponent,
     TableListComponent,
-    //AddRecruteursComponent,
+    AddRecruteursComponent,
+    AddResprhComponent,
+    ListRecruteursComponent,
+    ListResprhComponent,
     //IconsComponent,
     //MapsComponent,
     //NotificationsComponent,
Index: template/material-dashboard-angular2-master/src/app/layouts/admin-layout/admin-layout.component.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- template/material-dashboard-angular2-master/src/app/layouts/admin-layout/admin-layout.component.html	(revision 4a92d4a716e01abdfb387e01dd08072e9663dd94)
+++ template/material-dashboard-angular2-master/src/app/layouts/admin-layout/admin-layout.component.html	(date 1584620420020)
@@ -10,7 +10,7 @@
             <app-footer></app-footer>
         </div>
     </div>
-    <div class="fixed-plugin">
+    <!--<div class="fixed-plugin">
         <div class="dropdown show-dropdown open show">
             <a href="#" data-toggle="dropdown" aria-expanded="true">
                 <i class="fa fa-cog fa-2x"> </i>
@@ -74,7 +74,7 @@
             </ul>
         </div>
     </div>
-</div>
+</div>-->
 <!-- Buy-Modal-angular -->
 <div class="modal modal-angular fade" id="buy" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
 <div class="modal-dialog" role="document">
Index: template/material-dashboard-angular2-master/src/app/layouts/admin-layout/admin-layout.routing.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- template/material-dashboard-angular2-master/src/app/layouts/admin-layout/admin-layout.routing.ts	(revision 4a92d4a716e01abdfb387e01dd08072e9663dd94)
+++ template/material-dashboard-angular2-master/src/app/layouts/admin-layout/admin-layout.routing.ts	(date 1584619310377)
@@ -4,7 +4,10 @@
 import { UserProfileComponent } from '../../user-profile/user-profile.component';
 import { TableListComponent } from '../../table-list/table-list.component';
 import { AddRecruteursComponent } from '../../add-recruteur/add-recruteurs.component';
-//import { IconsComponent } from '../../icons/icons.component';
+import { ListRecruteursComponent } from '../../list-recruteurs/list-recruteurs.component';
+import { AddResprhComponent } from '../../add-resprh/add-resprh.component';
+import { ListResprhComponent } from '../../list-resprh/list-resprh.component';
+import { IconsComponent } from '../../icons/icons.component';
 //import { MapsComponent } from '../../maps/maps.component';
 //import { NotificationsComponent } from '../../notifications/notifications.component';
 //import { UpgradeComponent } from '../../upgrade/upgrade.component';
@@ -55,7 +58,12 @@
   //  { path: 'dashboard',      component: DashboardComponent },
     { path: 'user-profile',   component: UserProfileComponent },
     { path: 'table-list',     component: TableListComponent },
-   // { path: 'add-recruteur',     component: AddRecruteursComponent },
+    { path: 'add-recruteur',     component: AddRecruteursComponent },
+    { path: 'list-recruteurs',     component: ListRecruteursComponent },
+    { path: 'add-resprh',     component: AddResprhComponent },
+    { path: 'list-resprh',     component: ListResprhComponent },
+
+
     //{ path: 'icons',          component: IconsComponent },
     //{ path: 'maps',           component: MapsComponent },
     //{ path: 'notifications',  component: NotificationsComponent },
Index: template/material-dashboard-angular2-master/src/app/components/sidebar/sidebar.component.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- template/material-dashboard-angular2-master/src/app/components/sidebar/sidebar.component.ts	(revision 4a92d4a716e01abdfb387e01dd08072e9663dd94)
+++ template/material-dashboard-angular2-master/src/app/components/sidebar/sidebar.component.ts	(date 1584619679845)
@@ -11,7 +11,11 @@
    // { path: '/dashboard', title: 'Dashboard',  icon: 'dashboard', class: '' },
     { path: '/user-profile', title: 'Ajout Candidat',  icon:'person', class: '' },
     { path: '/table-list', title: 'Liste Des Candidats',  icon:'content_paste', class: '' },
-    //{ path: '/add-recruteur', title: 'Recruteur',  icon:'person', class: '' },
+    { path: '/add-recruteur', title: 'Ajout Recruteur',  icon:'person', class: '' },
+    { path: '/list-recruteurs', title: 'Liste Des Recruteurs',  icon:'content_paste', class: '' },
+    { path: '/add-resprh', title: 'Ajout Responsable RH',  icon:'person', class: '' },
+    { path: '/list-resprh', title: 'Liste Des Responsable RH',  icon:'content_paste', class: '' },
+
     //{ path: '/icons', title: 'Icons',  icon:'bubble_chart', class: '' },
     //{ path: '/maps', title: 'Maps',  icon:'location_on', class: '' },
     //{ path: '/notifications', title: 'Notifications',  icon:'notifications', class: '' },
